docs/source/rest.html.md.erb

```diff
@@ -95,7 +95,7 @@ You can pass two letters country codes ([ISO 639-1](https://en.wikipedia.org/wik
 Type: string
 </td>
 <td markdown="1">
-If specified, restrict the search results to a specific list of comma-separated countries. You can pass two letters country codes ([ISO 639-1](https://en.wikipedia.org/wiki/ISO_3166-1#Officially_assigned_code_elements)).
+If specified, restrict the search results to a specific list of comma-separated countries. You can pass two letters country codes ([ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1#Officially_assigned_code_elements)).
```

Comment:
@vvo Sorry for the late reply I missed the notification so we really have two different iso used. 

- For the language translations open street map uses ISO-639-1 and ISO-639-2 if it doesn't exist in the previous one (cf http://wiki.openstreetmap.org/wiki/Multilingual_names). 

- For the filtering per country, open street map provides a ISO-3166-1 (cf http://wiki.openstreetmap.org/wiki/Country_code)

So this change is wrong.